### ISSUE TITLE
Zip code field on home page doesn't do anything.

### ISSUE THREAD

**Issue by ResaWildermuth** [HTBox/allReady#193]
I'm able to input a zip code, but it doesn't respond to hitting enter or clicking the search button.

![image](https://cloud.githubusercontent.com/assets/15706343/11016942/6facb76c-8544-11e5-9ce6-b083cb49d8a9.png)

**Comment by ksk100** [comment]
To give a bit more info:

Hitting enter or pressing the search button both successfully submit a search to the API.  However the search API gives an error.

The problem is with `AllReadyDataAccessEF7.ActivitiesByPostalCode` which is throwing an `InvalidOperationException` with the message:

`InvalidOperationException: The Include operation is not supported when calling a stored procedure.`

The relevant stored procedure is `GetClosestActivitiesByPostalCode` (see it's call from L41 in `AllReadyDataAccessEF7.Activity.cs`.

**Comment by ksk100** [comment]
I'll take a look at this.

**Comment by ksk100** [comment]
This issue is pending on an answer to #306.

**Comment by cecilia-donnelly** [comment]
+1 on this.  The search now returns "There was an error.  Please try again later!" for valid and invalid zipcodes (at least all the ones I tested).  Thanks for investigating, @ksk100.

**Comment by tonysurma** [comment]
As a temp workaround can we remove the search box from view for this upcoming release as searching isn't key for private preview?  I will create a separate issue for that

**Comment by mheggeseth** [comment]
I think we could pretty easily replace the search box with a map of all the campaigns (Even relative to the user location).

**Comment by robrich** [comment]
The homepage no longer contains a zip code box.  Close this?
